doc/releases/changelog-dev.md

```diff
@@ -126,87 +179,36 @@
   tensor([0.89680614, 0.35281557, 0.80360155], requires_grad=True)
   ```
 
-* Added the new optimizer, `qml.SPSAOptimizer` that implements the simultaneous
-  perturbation stochastic approximation method based on
-  [An Overview of the Simultaneous Perturbation Method for Efficient Optimization](https://www.jhuapl.edu/SPSA/PDF-SPSA/Spall_An_Overview.PDF).
+<h4>The SPSA optimizer 🦾</h4>
```

Comment:
Does this deserve its own section? 🤔

Reply:
Maybe you can merge this section with the new quantum information quantity. I am not sure it deserves its own section indeed.

Reply:
I'm leaning towards just giving it a section. Just following what we've done in the past and I think it's consistent.